jupyter-book/linear_models/linear_models_wrap_up_quiz.md

```diff
@@ -241,10 +241,10 @@ On average, what is the increase in terms of accuracy by using the categorical
 features?
 
 - a) It gives similar results
-- b) It increases the statistical performance by 0.025
-- c) it increases the statistical performance by 0.05
-- d) it increases the statistical performance by 0.075
-- e) it increases the statistical performance by 0.1
+- b) It increases the generalization performance by 0.025
+- c) it increases the generalization performance by 0.05
+- d) it increases the generalization performance by 0.075
+- e) it increases the generalization performance by 0.1
```

Comment:
Since we are modifying here:

```suggestion
- c) It increases the generalization performance by 0.05
- d) It increases the generalization performance by 0.075
- e) It increases the generalization performance by 0.1
```